packages/components/src/ActionList/ActionList.component.js

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { Action } from '../Actions';
+import theme from './ActionList.scss';
+
+/**
+ * return the formatted action id
+ * if there is no action id, it is generated from the action label
+ * @param  {string} id        sidepanel id
+ * @param  {string} action    current action
+ * @return {string}            formatted id
+ */
+function getActionId(id, action) {
+	if (action.id || action.label) {
+		const actionId =
+			action.id ||
+			action.label
+				.toLowerCase()
+				.split(' ')
+				.join('-');
+		return id && `${id}-nav-${actionId}`;
+	}
+	return undefined;
+}
+
+function ActionListItem({ id, onSelect, action, isSelected }) {
+	const a11y = {
+		role: 'presentation',
+	};
+	const extra = {};
+
+	if (isSelected) {
+		// @see https://tink.uk/using-the-aria-current-attribute/
+		a11y['aria-current'] = true;
+	}
+	if (onSelect) {
+		extra.onClick = event => {
+			onSelect(event, action);
+			if (action.onClick) {
+				action.onClick(event);
+			}
+		};
+	}
+
+	const actionProps = Object.assign(
+		{},
+		action,
+		{
+			active: undefined, // active scope is only the list item
+			id: getActionId(id, action),
+			bsStyle: 'link',
+			role: 'link',
+		},
+		extra,
+	);
+
+	return (
+		<li
+			title={action.label}
+			key={action.key || action.label}
+			className={classNames(theme['tc-action-list-item'], 'tc-action-list-item', {
```

Comment:
Could be kind to accept extra classnames here in order to not break any QA automated tests!